verl/trainer/config/sft_trainer.yaml

```diff
@@ -38,6 +38,7 @@ optim:
   weight_decay: 0.01
   warmup_steps_ratio: 0.1
   clip_grad: 1.0
```

Comment:
could you update config.rst with the new added option? thx

Reply:
Does this works?

https://github.com/volcengine/verl/pull/1041/commits/725289e6377ecefcc1aff6811b585fa0c4f60b9e